nautobot/core/settings.py

```diff
@@ -784,6 +784,9 @@
 # If enabled, a `task-sent` event will be sent for every task so tasks can be tracked before they're consumed by a worker.
 CELERY_TASK_SEND_SENT_EVENT = True
 
+# Use threads Celery pool - required for git repository job support
+CELERY_WORKER_POOL = "threads"
```

Comment:
Definitely needs a `.changed` changelog entry.

Reply:
Does this fix #5063?

Don't we need to do the testing in #5164 before we proceed in this direction?